src/tree/localProject/LocalFunctionsTreeItem.ts

```diff
@@ -87,11 +87,11 @@ export class LocalFunctionsTreeItem extends FunctionsTreeItemBase {
                 rejectUnauthorized: hostRequest.rejectUnauthorized
             });
             return await this.createTreeItemsWithErrorHandling(
-                <WebSiteManagementModels.FunctionEnvelope[]>functions.parsedBody,
+                <FunctionEnvelope[]>functions.parsedBody,
                 'azFuncInvalidLocalFunction',
                 async func => {
                     func = requestUtils.convertToAzureSdkObject(func);
-                    return LocalFunctionTreeItem.create(context, this, nonNullProp(func, 'name'), func.config, undefined, func);
+                    return LocalFunctionTreeItem.create(context, this, nonNullProp(func, 'name'), new ParsedFunctionJson(func.config), undefined, func);
```

Comment:
So the typing here changed to  Record<string, unknown> which is basically just a Set which is basically represented as an object.  Passing in that data type to ParsedFunctionJson() seems to create the JSON object that we need for local tree items, which is why I ended up just creating it rather than changing the tree items to accept a Record<string, unknown> typing.